app/views/diary/steps/_select_event.html.erb

```diff
@@ -6,4 +6,4 @@
   <%= f.govuk_radio_button :event, options[:emotional_health],  label: { text: t("select_event.event_options.emotional_health") }, link_errors: true %>
 <% end %>
 
-<%= f.text_field :placement_id, pattern: "[0-9]*", inputmode: "numeric", readonly: true, hidden: false, value: params[:placement_id] || f.object.placement_id%>
+<%= f.text_field :placement_id, pattern: "[0-9]*", inputmode: "numeric", readonly: true, hidden: true, value: params[:placement_id] || f.object.placement_id%>
```

Comment:
If you revert this change 🙏  then this PR will not conflict with the branch I'm about to merge. I'm removing this line entirely anyway, so it's safe for you to do.